doc/how_to/index.rst

```diff
@@ -12,3 +12,4 @@ Guides on how to solve specific, short problems in SpikeInterface. Learn how to.
     load_matlab_data
     combine_recordings
     process_by_channel_group
+    make_a_sorting
```

Comment:
As a title of the section "make a sorting" sounds to me like how to make a new sorting extractor programatically.

maybe "load your own sorting data to spikeinterface"  or something else?

Reply:
Do you want that for the toctree? The actual title isn't influenced by the file name at all. Are you worried we won't know which one is which in the future? I could definitely change but didn't want to be over verbose just for making the index.

Reply:
Yes for me, I agree to change the title itself as I was not 100% sure what `Make your own Sorting` referred to. For the toctree / .rst filename `make_a_sorting` I am okay with an abbreviated version of the title.